index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Cherry on Top - Recipe Input Form</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="icon" href="favicon.ico" type="image/x-icon">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  </head>
+  <body>
+
+  <div>
+    <header>
+        <img src="./images//icon/icon-pie.png" alt="Cherry on Top" class="cherry-logo">
+         <h1>Cherry on Top</h1>
+         <h3>Making Life Sweeter One Bite at a Time</h3>
+    </header>
+  </div>
+
+    <div>
+      <h2>Submit your own recipe</h2>
+          <p>Spill the secret and help making the world a sweeter place!</p> 
+      </div>  
+
+    <form action="http://httpbin.org/anything" method="POST">
+    <h4>Contact Details</h4>
+      <label for="Name">Name:</label>
+      <input required autofocus type="text" placeholder="Cherry Garcia"><br>
+      <label for="Email">E-mail:</label>
+      <input required type="email" name="" id="email" placeholder="hello@email.com"><br>
```

Comment:
When we have many attributes, we can make it more readable by splitting them into multiple lines:
```
<input
  required
  type="email"
  name=""
  id="email"
  placeholder="hello@email.com">
```